templates/telemetry/core_clients_daily_v1/query.sql

```diff
@@ -1,23 +1,14 @@
 WITH
```

Comment:
is this supposed to move to `templates/telemetry_derived/core_clients_daily_v1/query.sql`?

Reply:
or perhaps it's that it's in both places and just wasn't removed?

Reply:
i like the one in `templates/telemetry_derived/core_clients_daily_v1/query.sql` better

Reply:
Yes, this looks like a mistake. This should have been replaced with a view.

Reply:
> i like the one in `templates/telemetry_derived/core_clients_daily_v1/query.sql` better

Why? You prefer explicitly listing `DATE(submission_timestamp)` instead of `submission_date` every time?

Reply:
idk, i can't see them both to compare anymore. probably nothing significant, i blame the comment on morning brain.